Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -345,6 +346,11 @@ void RenderBox::styleDidChange(StyleDifference diff, const RenderStyle* oldStyle
         }
     }
 
+    if (layer() && oldStyle && oldStyle->shouldPlaceVerticalScrollbarOnLeft() != newStyle.shouldPlaceVerticalScrollbarOnLeft()) {
+        if (auto* scrollableArea = layer()->scrollableArea())
+            scrollableArea->scrollbarsController().rtlStyleDidChange(shouldPlaceVerticalScrollbarOnLeft() ? UserInterfaceLayoutDirection::RTL : UserInterfaceLayoutDirection::LTR);
```

Comment:
`rtlStyleDidChange` would be better named as `scrollbarLayoutDirectionChanged`.

Reply:
Fixed.